lib/shared/bucketing-assembly-script/project.json

```diff
@@ -30,10 +30,10 @@
             "options": {
                 "commands": [
                     {
-                      "command": "asc assembly/index.ts --target debug --exportRuntime --initialMemory 100 --maximumMemory 10000 --bindings esm"
+                      "command": "asc assembly/index.ts --target debug --exportRuntime --initialMemory 10000 --maximumMemory 20000 --bindings esm"
```

Comment:
@JamieSinn / @ajwootto Do we still need the initial memory to be this high?

Reply:
yup otherwise the tests won't pass

Reply:
can we just set it for the debug build then?

Reply:
It'll be required for any sort of use at scale in production too

Reply:
but does it need 1.2GB? (20,000 * 64kb pages)

Reply:
20k is the initial load - if that's not used the GC will free them and scale up as needed.